### DISCUSSION TITLE
How to specify the connection to a specific database?

### DISCUSSION THREAD

**Discussion by garik-pilot** [neo4j-contrib/neovis.js#324] [Q&A]
Hi, I reviewed all examples in  neovis package to figure out how to configure the connection to a database like (from example-labels.html): 
					serverUrl: "bolt://54.209.44.254:33146",
					serverUser: "neo4j",
					serverPassword: "investigators-spill-future" 

How I can point to a specific database at serverUrl: "bolt://54.209.44.254:33146"  if I have multiple instances are running in neo4j?
 May be it is better to externalize the db connection string like: 
 
export const dbConfg = {
    scheme: 'neo4j+s',
    host: '54.209.44.254',
    port: 33146,
    database: 'myneo4jdb',
    username: 'readonly',
    password: 'readonly',
};

**Comment by thebestnom** [comment]
Can the driver do that? In that case you can pass an driver instance instead of sending config

**Reply by garik-pilot** [reply]
we also use another tool https://docusaurus.io/ that allows to do that.

**Reply by thebestnom** [reply]
What a production deploy tool has to do with neo4j driver? 😅

**Reply by garik-pilot** [reply]
it seems like a db name can be specified explicitly when I look at neovis.ts but it is not added to the examples.   

		this.#config = config;
		this.#driver = isNeo4jDriver(config.neo4j) ? config.neo4j : Neo4j.driver(
			config.neo4j?.serverUrl ?? defaults.neo4j.neo4jUri,
			Neo4j.auth.basic(
				config.neo4j?.serverUser ?? defaults.neo4j.neo4jUser,
				config.neo4j?.serverPassword ?? defaults.neo4j.neo4jPassword
			),
			deepmerge(defaults.neo4j.driverConfig, config.neo4j?.driverConfig ?? {})
		);
		**this.#database = **config.serverDatabase**;**
		this.#query = config.initialCypher ?? defaults.neo4j.initialQuery;
		this.#container = document.getElementById(config.containerId);
	}

**Reply by thebestnom** [reply]
Oh, I thought you meant server list with primary secondary urls like mongo 😅
Yeah you can and it is in the d.ts
I can add one to an example if it is not enough

**Comment by garik-pilot** [comment]
It would be great 👍
